frontend/src/app/testResults/viewResults/actionMenuModals/TestResultDetailsModal.test.tsx

```diff
@@ -78,8 +86,17 @@ syphilisTestResult.results = [
     __typename: "MultiplexResult",
   },
 ];
-syphilisTestResult.pregnancy = "77386006";
-syphilisTestResult.symptoms = '{"266128007":"true"}';
+syphilisTestResult.surveyData.pregnancy = "77386006";
+syphilisTestResult.surveyData.syphilisHistory = "1087151000119108";
+syphilisTestResult.surveyData.symptoms =
+  '{"91554004":"false","15188001":"false","724386005":"false","56940005":"true","266128007":"true","246636008":"true","68225006":"false","195469007":"false","26284000":"false"}';
```

Comment:
nit: I don't think this will change the tests but for consistency can we make the values match the [symptoms map defined here](https://github.com/CDCgov/prime-simplereport/blob/00710d95f297e65a30bdf3b869e1915e1f906ba0/frontend/src/patientApp/timeOfTest/constants.ts#L90)

I probably should have removed the commented out symptoms when I was working on the other PR 😅 (Not sure why we were keeping them before 🤔)